conf/janelia.config

```
@@ -0,0 +1,30 @@
+params {
+    config_profile_contact     = 'Konrad Rokicki (rokickik@janelia.hhmi.org)'
+    config_profile_name        = 'Janelia Compute Cluster'
+    config_profile_description = 'IBM Platform LSF Cluster at Janelia Research Campus'
+    config_profile_url         = 'https://www.janelia.org'
+
+    max_cpus   = 48
+    max_memory = '768.GB'
+    max_time   = '48.h'
+
+    lsf_opts = ''
+    schema_ignore_params = 'lsf_opts'
```

The way I implemented this was crap, annoying this will wipe out the defaults. Instead could you please copy the default string and add in your extra param. Sorry..

```suggestion
    schema_ignore_params = 'genomes,lsf_opts'
```